ops/gulp/dev-watch.js

```diff
@@ -21,6 +21,9 @@ const task = () => {
 	//update styleguide styles
 	gulp.watch([`./docs/styleguide/styles/*.scss`], ['docs-styleguide']);
 
+	//watch docs js
+	gulp.watch([`./src/scripts/*.js`,`./ops/gulp/*.js`], ['docs-js']);
```

Comment:
No need to watch `./ops/gulp/*.js`. This would run `docs-js` when any ops JS file is modified and most of the JS files are unrelated to the site JS.

Also, the watch task here will compile the JS but will not update the browser automatically. See `dev-serve.js` where Browsersync file watching is specified to inject the JS into the page when running the local server.